src/index.js

```diff
@@ -509,6 +585,7 @@ const _registering = (context, event) => async (cbParent) => {
 }
 
 const _readSessionInfoIntoState = async (context, event) => {
+  console.log('[_readSessionInfoIntoState]')
```

Comment:
debug ?

Reply:
will be removed in a future PR to xstate branch :+1: